test/test_transforms.py

```diff
@@ -135,72 +219,57 @@ def test_center_crop_2(self):
-        to_pil_image = transforms.ToPILImage()
-        h = random.randint(5, 25)
-        w = random.randint(5, 25)
-        for single_dim in [True, False]:
-            crop_h = random.randint(1, h)
-            crop_w = random.randint(1, w)
-            if single_dim:
-                crop_h = min(crop_h, crop_w)
-                crop_w = crop_h
-                transform = transforms.FiveCrop(crop_h)
-            else:
-                transform = transforms.FiveCrop((crop_h, crop_w))
-
-            img = torch.FloatTensor(3, h, w).uniform_()
-            results = transform(to_pil_image(img))
-
-            self.assertEqual(len(results), 5)
-            for crop in results:
-                self.assertEqual(crop.size, (crop_w, crop_h))
-
-            to_pil_image = transforms.ToPILImage()
-            tl = to_pil_image(img[:, 0:crop_h, 0:crop_w])
-            tr = to_pil_image(img[:, 0:crop_h, w - crop_w:])
-            bl = to_pil_image(img[:, h - crop_h:, 0:crop_w])
-            br = to_pil_image(img[:, h - crop_h:, w - crop_w:])
-            center = transforms.CenterCrop((crop_h, crop_w))(to_pil_image(img))
-            expected_output = (tl, tr, bl, br, center)
-            self.assertEqual(results, expected_output)
 
-    def test_ten_crop(self):
-        to_pil_image = transforms.ToPILImage()
-        h = random.randint(5, 25)
-        w = random.randint(5, 25)
-        for should_vflip in [True, False]:
-            for single_dim in [True, False]:
-                crop_h = random.randint(1, h)
-                crop_w = random.randint(1, w)
-                if single_dim:
-                    crop_h = min(crop_h, crop_w)
-                    crop_w = crop_h
-                    transform = transforms.TenCrop(crop_h,
-                                                   vertical_flip=should_vflip)
-                    five_crop = transforms.FiveCrop(crop_h)
-                else:
-                    transform = transforms.TenCrop((crop_h, crop_w),
-                                                   vertical_flip=should_vflip)
-                    five_crop = transforms.FiveCrop((crop_h, crop_w))
-
-                img = to_pil_image(torch.FloatTensor(3, h, w).uniform_())
-                results = transform(img)
-                expected_output = five_crop(img)
-
-                # Checking if FiveCrop and TenCrop can be printed as string
+def test_center_crop():
+    height = random.randint(10, 32) * 2
+    width = random.randint(10, 32) * 2
+    oheight = random.randint(5, (height - 2) / 2) * 2
+    owidth = random.randint(5, (width - 2) / 2) * 2
+
+    img = torch.ones(3, height, width)
+    oh1 = (height - oheight) // 2
+    ow1 = (width - owidth) // 2
+    imgnarrow = img[:, oh1:oh1 + oheight, ow1:ow1 + owidth]
+    imgnarrow.fill_(0)
+    result = transforms.Compose([
+        transforms.ToPILImage(),
+        transforms.CenterCrop((oheight, owidth)),
+        transforms.ToTensor(), ])(img)
+    assert_equal(result.sum(), 0,
```

Comment:
as this is just an integer comparison, we should just use `assert a == b` as per the instructions in https://github.com/pytorch/vision/issues/3945#issue-909493806

Also I see above that the formatting is a bit odd:

```py
         transforms.ToTensor(), ])(img)
```

Please make sure to preserve the original formatting :)